web/modules/custom/joinup_news/config/install/core.entity_form_display.node.news.default.yml

```diff
@@ -53,7 +52,6 @@ content:
     third_party_settings: {  }
 hidden:
   created: true
-  field_news_parent: true
```

Comment:
Nope. Actually this was the entire ticket :) Previously, we already had implemented the notion of a news entity belonging to a solution, but since the solution was not an Og, a normal reference was needed. Now, the field itself is obsolete. The solution is normally referenced by og_group_ref field.